evap/staff/tests/test_forms.py

```diff
@@ -3,13 +3,34 @@
 from django.test import TestCase
 from model_bakery import baker
 
-from evap.evaluation.models import (Contribution, Course, Degree, EmailTemplate, Evaluation, Question,
-                                    Questionnaire, Semester, UserProfile)
-from evap.evaluation.tests.tools import (create_evaluation_with_responsible_and_editor, get_form_data_from_instance,
-                                         to_querydict)
-from evap.staff.forms import (ContributionForm, ContributionCopyForm, ContributionFormSet, CourseForm,
-                              EvaluationEmailForm, EvaluationForm, EvaluationCopyForm,
-                              QuestionnaireForm, SingleResultForm, UserForm)
+from evap.evaluation.models import (
+    Contribution,
+    Course,
+    Degree,
+    EmailTemplate,
+    Evaluation,
+    Question,
+    Questionnaire,
+    Semester,
+    UserProfile,
+)
+from evap.evaluation.tests.tools import (
+    create_evaluation_with_responsible_and_editor,
+    get_form_data_from_instance,
+    to_querydict,
+)
+from evap.staff.forms import (
+    ContributionForm,
+    ContributionCopyForm,
+    ContributionFormSet,
+    CourseForm,
+    EvaluationEmailForm,
+    EvaluationForm,
+    EvaluationCopyForm,
+    QuestionnaireForm,
+    SingleResultForm,
+    UserForm,
+)
```

Comment:
@janno42 commented on this -- I think there is some kind of flag that allows it to keep multiple imported classes on one line?

Reply:
Ahh, no -- https://black.readthedocs.io/en/stable/compatible_configs.html

isort supports out "old" formatting. It seems like black does not.

I'm fine with this formatting.